frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -2851,6 +2851,29 @@ static bool resolveFnCallSpecial(Context* context,
   // TODO: .borrow()
   // TODO: chpl__coerceCopy
 
+  // explicit param casts are resolved here
+  if (ci.name() == USTR(":")) {
+    auto lhs = ci.actual(0).type();
+    auto rhs = ci.actual(1).type();
```

Comment:
IMO lhs and rhs are a bit unintuitive when talking about a cast. I'd prefer `fromValue` and `toType` or just `from` and `to`.